docs/reference/patches.rst

```diff
@@ -0,0 +1,329 @@
+.. _patches:
```

Comment:
Wow! I didn't even realize you included all the patches here until now. This is a great start to fully documenting all the patches. 🎉